pkg/webhooks/hiveextension/v1beta1/agentclusterinstall_mutating_hook.go

```diff
@@ -115,8 +115,10 @@ func (a *AgentClusterInstallMutatingAdmissionHook) SetDefaults(admissionSpec *ad
 
 	// if UserNetworkManagement is not set by the user apply the defaults:
 	// true for SNO and false for multi-node
-	if newObject.Spec.Networking.UserManagedNetworking == nil {
-		patchList = append(patchList, patchUserManagedNetworking(newObject, contextLogger))
+	if !installAlreadyStarted(newObject.Status.Conditions) || !newObject.DeletionTimestamp.IsZero() {
```

Comment:
Shouldn't this be `&&`? We need to ensure that installation didn't start **and** deletion is not in-progress.

Reply:
Also, we want `DeletionTimestamp.IsZero()` to be `true`, not `false`.